From 55b0bc236e7b2b20abbcb01fe6f929a127cba381 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Tue, 9 Apr 2013 14:00:38 -0700
Subject: [PATCH 1/5] aufs: kbuild patch

Integrating aufs-kbuild patch from:

   git://aufs.git.sourceforge.net/gitroot/aufs/aufs3-standalone.git

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 fs/Kconfig                |    1 +
 fs/Makefile               |    1 +
 include/uapi/linux/Kbuild |    1 +
 3 files changed, 3 insertions(+)

diff --git a/fs/Kconfig b/fs/Kconfig
index d6e0aaf..da05819 100644
--- a/fs/Kconfig
+++ b/fs/Kconfig
@@ -212,6 +212,7 @@ source "fs/sysv/Kconfig"
 source "fs/ufs/Kconfig"
 source "fs/exofs/Kconfig"
 source "fs/f2fs/Kconfig"
+source "fs/aufs/Kconfig"
 
 endif # MISC_FILESYSTEMS
 
diff --git a/fs/Makefile b/fs/Makefile
index a0c4148..d91e380 100644
--- a/fs/Makefile
+++ b/fs/Makefile
@@ -128,3 +128,4 @@ obj-y				+= exofs/ # Multiple modules
 obj-$(CONFIG_CEPH_FS)		+= ceph/
 obj-$(CONFIG_PSTORE)		+= pstore/
 obj-$(CONFIG_YAFFS_FS)		+= yaffs2/
+obj-$(CONFIG_AUFS_FS)           += aufs/
diff --git a/include/uapi/linux/Kbuild b/include/uapi/linux/Kbuild
index 4e67194..cc5b423 100644
--- a/include/uapi/linux/Kbuild
+++ b/include/uapi/linux/Kbuild
@@ -56,6 +56,7 @@ header-y += atmppp.h
 header-y += atmsap.h
 header-y += atmsvc.h
 header-y += audit.h
+header-y += aufs_type.h
 header-y += auto_fs.h
 header-y += auto_fs4.h
 header-y += auxvec.h
-- 
1.7.10.4

